src/components/FilterBar/SavedSearchesList/SavedSearchesList.react.js

```diff
@@ -0,0 +1,57 @@
+import {SavedSearchesListItem} from './SavedSearchesListItem.react';
+
+export class SavedSearchesList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = this.getStateFromStores();
+    this.props.filterBarStore.addChangeListener(this._onChange.bind(this));
```

Comment:
this should go in componentDidMount just in cause an event is triggered before it has mounted.